### PR TITLE
New/404

### PR DESCRIPTION
**Pull request by AbhinavThukral97**
Error Page Implementation

### CODE REVIEW

**Comment by lfaucon** [comment]
@AbhinavThukral97 Let's merge the conflicts and merge this PR first as it is a part of #1882

File: frog/imports/ui/ErrorPage/index.stories.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import ErrorPage from '.';
+
+storiesOf('Error Page', module).add('Graph', () => (
+  <PageNotFound title="404 Graph" message="This graph does not exist" />
```
Comment (lfaucon) [inline frog/imports/ui/ErrorPage/index.stories.js]: `PageNotFound` is not defined. I think you meant to use the component `ErrorPage`

**Review by lfaucon — COMMENTED** [review]
Looks good!

File: frog/imports/ui/ErrorPage/index.stories.js (hunk 1 above)
Comment (AbhinavThukral97) [reply frog/imports/ui/ErrorPage/index.stories.js]: Ah yes! My bad.

**Review by lfaucon — APPROVED** [review]

**Pull request merged**